lib/components/Dropdown.js

```diff
@@ -48,12 +49,14 @@ export default class Dropdown extends Component {
             icon={icon || "ri-arrow-down-s-line"}
             iconPosition="right"
             disabled={disabled}
+            data-cy={`${hyphenize(label)}-dropdown-icon`}
```

Comment:
@datt _a If the button style is icon, we are not expecting the label prop. Hence we need a fallback when label prop is not provided. 
It can be handled in `hyphenize` function or as `label || "dropdown"`

Same is the case for Select component. Please check.

Reply:
Check https://github.com/bigbinary/neeto-ui/blob/master/lib/components/Input.js#L54 where I had to give fallback condition.

Reply:
Except in the `Label` component, the label prop is optional.